### PR TITLE
Improve (slightly) the grammar in our Airflow integration docs

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

File: docs/integrations/airflow.md [hunk 1]
```diff
@@ -9,30 +9,38 @@ redirect_from: ../using/airflow.html
...
+
+## Create a lakeFS connection on Airflow
 
-## Creating the lakeFS connection
-For authenticating to the lakeFS server, you need to create a new [Airflow Connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html)
-of type HTTP and pass it to your DAG. You can do that using the Airflow UI or the cli.
-Here’s an example Airflow command that does just that:
+To access the lakeFS server and authenticate with it, create a new [Airflow
+Connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html)
+of type HTTP and pass it to your DAG. You can do that using the Airflow UI
```
Comment (adipolak) [inline docs/integrations/airflow.md:21]: what about: "pass it to your airflow DAG" ?

File: docs/integrations/airflow.md [hunk 2]
```diff
@@ -9,30 +9,38 @@ redirect_from: ../using/airflow.html
...
+or the cli.  Here’s an example Airflow command that does just that:
```
Comment (adipolak) [inline docs/integrations/airflow.md:22]: should cli be in caps? CLI or is it usually  cli

File: docs/integrations/airflow.md [hunk 3]
```diff
@@ -9,30 +9,38 @@ redirect_from: ../using/airflow.html
...
 
 ```bash
 airflow connections add conn_lakefs --conn-type=HTTP --conn-host=http://<LAKEFS_ENDPOINT> \
- --conn-extra='{"access_key_id":"<LAKEFS_ACCESS_KEY_ID>","secret_access_key":"<LAKEFS_SECRET_ACCESS_KEY>"}'
+    --conn-extra='{"access_key_id":"<LAKEFS_ACCESS_KEY_ID>","secret_access_key":"<LAKEFS_SECRET_ACCESS_KEY>"}'
 ```
 
-## Installing lakeFS Airflow package
-Installing the package using `pip`:
```
Comment (adipolak) [inline docs/integrations/airflow.md:27]: please add the package link - https://github.com/treeverse/airflow-provider-lakeFS

**Review by adipolak — CHANGES_REQUESTED** [review]
Thanks, Ariel, looks great, notice the couple of comments to address. After it, it's good to go!

File: docs/integrations/airflow.md (hunk 1 above)
Comment (arielshaqed) [reply docs/integrations/airflow.md]: Good point, I wrote it backwards.  Going with "add it to your DAG": the command _literally_ says "airflow connection(s) add".

File: docs/integrations/airflow.md (hunk 3 above)
Comment (arielshaqed) [reply docs/integrations/airflow.md:27]: Added a package link; I think it is https://pypi.org/project/airflow-provider-lakefs/ so I added that link.  Users should care less about where the package is built, and more about from where to download it.

**Review by arielshaqed — COMMENTED** [review]
Thanks!  All good points, I took the liberty of phrasing things a bit differently from your suggestions.

File: docs/integrations/airflow.md (hunk 3 above)
Comment (adipolak) [reply docs/integrations/airflow.md:27]: good point!

**Review by adipolak — APPROVED** [review]
Looks great! Approved!

**Comment by arielshaqed** [comment]
Thanks!

Pulling by the powers vested in all of us as Administrators - Nesdie test failure is irrelevant to docs.

**Pull request merged**